components/bluetooth_service/a2dp_stream.c

```diff
@@ -614,4 +617,12 @@ esp_err_t periph_bt_volume_down(esp_periph_handle_t periph)
 }
 #endif
 
+esp_err_t periph_bt_get_connected_bd_addr(esp_periph_handle_t periph, uint8_t *dest)
+{
+    if (!dest)
+        return ESP_ERR_INVALID_ARG;
```

Comment:
How about BT is not connected? Maybe ESP_ERR_INVALID_STATE show be returned.

Reply:
That sounds good. I have added a connected_flag which is set to 0 until connected, and will return ESP_ERR_INVALID_STATE if this function is called when connected_flag is 0.